app/javascript/src/Policies/styles/policies.scss

```diff
@@ -63,6 +64,25 @@ $subtleColor: #ddd;
     }
   }
 
+  .PolicyConfiguration--preview {
+    overflow: hidden;
+    position: relative;
+
+    &::after {
+      color: $subtleColor; //$label-color;
+      content: 'Preview';
+      font-size: 10rem;
```

Comment:
Shouldn't we use `line-height-times()` for font size, margins, paddings, etc ?

Reply:
Yes, but those variables are not available here.  We'd have to include them… Hello React.

Reply:
Ah.. oops you're right